Core/src/org/sleuthkit/autopsy/directorytree/ViewContextAction.java

```diff
@@ -168,176 +169,223 @@ public ViewContextAction(String displayName, Content content) {
     public void actionPerformed(ActionEvent event) {
         EventQueue.invokeLater(() -> {
 
-            /*
-             * Get the parent content for the content to be selected in the
-             * results view. If the parent content is null, then the specified
-             * content is a data source, and the parent tree view node is the
-             * "Data Sources" node. Otherwise, the tree view needs to be
-             * searched to find the parent treeview node.
-             */
-            Content parentContent = null;
-            try {
-                parentContent = content.getParent();
-            } catch (TskCoreException ex) {
-                MessageNotifyUtil.Message.error(Bundle.ViewContextAction_errorMessage_cannotFindDirectory());
-                logger.log(Level.SEVERE, String.format("Could not get parent of Content object: %s", content), ex); //NON-NLS
-                return;
-            }
-            
-            if ((parentContent != null) 
-                    && (parentContent instanceof UnsupportedContent)) {
+            Content parentContent = getParentContent(this.content);
+
+            if ((parentContent != null) && (parentContent instanceof UnsupportedContent)) {
                 MessageNotifyUtil.Message.error(Bundle.ViewContextAction_errorMessage_unsupportedParent());
                 logger.log(Level.WARNING, String.format("Could not navigate to unsupported content with id: %d", parentContent.getId())); //NON-NLS
-                return;
```

Comment:
This seems like a return we would want since creating the parentTreeNode and settingNodeSelection does not seem to have much purpose if can't navigate to the node, or am I missing something?